src/OnlineSales/Infrastructure/PluginManager.cs

```diff
@@ -1,111 +1,112 @@
-﻿// <copyright file="PluginManager.cs" company="WavePoint Co. Ltd.">
```

Comment:
Same here, do you know why does it look like you delete the whole file and re-created it again? It is hard to see real changes this way.